build.py

```diff
@@ -67,17 +67,18 @@
 # incorrectly load the other version of the openvino libraries.
 #
 TRITON_VERSION_MAP = {
-    '2.37.0dev': (
-        '23.08dev',  # triton container
-        '23.06',  # upstream container
-        '1.15.0',  # ORT
-        '2023.0.0',  # ORT OpenVINO
-        '2023.0.0',  # Standalone OpenVINO
-        '2.4.7',  # DCGM version
-        'py310_23.1.0-1')  # Conda version.
+    "2.37.0dev": (
+        "23.08dev",  # triton container
+        "23.06",  # upstream container
+        "1.15.0",  # ORT
+        "2023.0.0",  # ORT OpenVINO
+        "2023.0.0",  # Standalone OpenVINO
+        "2.4.7",  # DCGM version
+        "py310_23.1.0-1",
+    )  # Conda version.
```

Comment:
This still looks wrong, I wonder if it's not possible to have a comment on the last item of the tuple in the rules?

Reply:
Longer term improvement that could fix this would be to change the tuple to a nested dict where the keys reflect the versions. I think that will clarify the usage throughout the code as well.

Instead of 
```python
# Access ORT version
TRITON_VERSION_MAP[version][2]
```
it could be:
```python
TRITON_VERSION_MAP = {
    "2.37.0dev": {
        "triton_container": "23.08dev",
        "upstream_container": "23.06",  
        "onnxruntime": "1.15.0", 
    # ...
}

TRITON_VERSION_MAP[version]["onnxruntime"]
```

Reply:
100% agree with that recommendation! Really like the improvement, and I know @nnshah1 and @nv-kmcgill53 are currently looking into build.py improvements, so this could be a great direction.

This wasn't a pre-commit hook issue so much as the easiest/safest way to rebase being to use the latest file (in this case `build.py`) then re-run pre-commit on it. I try to integrate any previously-applied changes/feedback in, but I must have missed this one. Thanks for catching this, Ryan!

Reply:
LMK if there are any tickets on build.py improvements and I can link this comment or something.

Reply:
I was going to use today to write up that proposal but got bogged down on other priorities.